MoreLinq.Test/ZipLongestTest.cs

```diff
@@ -58,21 +58,6 @@ public void ZipLongestIsLazy()
             bs.ZipLongest(bs, BreakingFunc.Of<int, int, int>());
         }
 
-        [Test]
-        public void ZipLongestDisposeSequencesEagerly()
```

Comment:
Removed as simpler code does not dispose immediately, but instead keeps calling `.MoveNext()` expecting `false` return values. Is there a specific advantage to disposing early?